policy/README.md

```diff
@@ -365,6 +384,130 @@ If a vehicle is matched with a rule, then it _will not_ be considered in the sub
+| `omf_review_date`            | [timestamp][ts] | Optional | If `omf_review`, add timestamp. E.g. "1611958749" |
+| `agency_uuid`                | UUID            | Required | UUID of the agency this file applies to. Must come from [agencies.csv](/agencies.csv) file. E.g. "737a9c62-c0cb-4c93-be43-271d21b784b5" |
+| `agency_name`                | text            | Required | Name of the agency this file applies to. E.g. "Louisville Metro" |
+| `agency_time_zone`           | text            | Required | Timezone used for dates and times across all MDS endpoints. E.g. "America/New_York" |
+| `agency_currency`            | text            | Required | Currency used for all monetary values across all MDS endpoints. E.g. "USD" |
+| `agency_policy_website_url`  | URL             | Required | URL of the agency's transportation policy page. E.g. "https://www.cityname.gov/transporation/shared-devices.htm" |
+| `agency_policy_document_url` | URL             | Optional | URL of the agency's operating permit rules that mention data requirements. E.g. "https://www.cityname.gov/mds_data_policy.pdf" |
+| `gbfs_required`              | Enum            | Required | yes/no. Is public GBFS required explicitly by providers? E.g. "yes" |
+| `url`                        | URL             | Required | URL of this file. E.g.  "https://mds.cityname.gov/requirements/1.2.0" |
+
+[Top][toc]
+
+#### Requirement MDS Version
+
+Contains a list of providers and APIs/endpoints/fields that a version of MDS applies to in its [Requirement](#requirement) data feed. Unique combinations for MDS versions and specific providers can be defined. For example an agency can devine MDS version 1.2.0 for Provider #1 in a pilot with beta endpoints and optional fields, version 1.2.0 for other providers without beta features, and version 1.1.0 for Provider #2 with docked bikeshare. 
+
+```jsonc
+// ...  
+  "mds_versions": [
+    {
+      "version" : "[MDS VERSION NUMBER]",
+      "provider_ids": [
+        "[PROVIDER UUID]",
+        "[PROVIDER UUID]"
+      ],
+      "mds_apis" [
+        {
+          // ...
+        },
+        // other MDS APIs
+      ]
+    }
+  ]
+// ...
+```
+
+| Name                         | Type           | Required / Optional | Description              | 
+| ---------------------------- | -------------- | -------- | ----------------------------------- | 
+| `version`                    | text           | Required | Version number of an official MDS release | 
+| `provider_ids`               | UUID[]         | Required | Array of providers that apply to this part of the requirements | 
+
+[Top][toc]
+
+#### Requirement MDS APIs
+
+For each combination of MDS version and provider list, you can specify the MDS APIs, endpoints, and optional fields that are required per your agency's policy. This is an array within the [Requirement MDS Version](#requirement-mds-version) `mds_apis` section in the [Requirement](#requirement) data feed.
```

Comment:
I think we will want support for `modes` and some SLA fields such as latency.  E.g. "/events must arrive within 30 seconds."

Reply:
I think SLA's and what they entail may come in a the next version of this. Unless you have some specific proposals that we can look at and agree on. I see a lot of variability here that could be challenging.

But your mention of modes makes me think maybe modes could be listed for each Release/Provider/state/end combination too, so different modes could apply to the same provider, or different MDS versions...

Reply:
Thinking about the SLA some more... Maybe there could be a field for 'expected_latency' at the endpoint level which specified an ISO interval that is the minimum expected time that an agency would like to get the data. Eg 5 minutes if they want near real time data, but could be 24 hours if they only have use cases that require it to be updated once per day? Not sure how that sits with certain endpoint latency as required in the spec already, and if this would be allowed to override that.

Reply:
I think SLA should wait for a future version of this, so let's put a pin in it.  I did add `vehicle_types` [here](https://github.com/openmobilityfoundation/mobility-data-specification/commit/6c6a89d7ed24f220eaa6607caf0efcd7a853aa4e) so you can optionally specify an array of types that apply to the specific version and providers in question. Does this solve your mode idea?